src/modules/audit/audit.controller.ts

```diff
@@ -22,6 +23,8 @@ import { UpdateAuditDto, UpdateAuditSchema } from './dtos/update-audit.dto';
 @ApiTags('audits')
 @Controller({ path: 'audits', version: '1' })
 export class AuditController {
+  private readonly logger = new Logger(AuditController.name); // Initialize logger for the controller
```

Comment:
Please, can you create a module for this? and use injection for instance:

` private readonly logger: Logger, // instantiate logger`

```
import { Logger, Module } from '@nestjs/common';
import { ConfigModule } from '@nestjs/config';
import { UserModule } from './user/user.module';

@Module({
  imports: [
    UserModule,
  ],
  providers: [Logger],
})

export class AppModule {}

```